api/Elementa.api

```diff
@@ -1274,7 +1284,7 @@ public final class gg/essential/elementa/constraints/AdditiveConstraint : gg/ess
 }
 
 public final class gg/essential/elementa/constraints/AlphaAspectColorConstraint : gg/essential/elementa/constraints/ColorConstraint {
-	public fun <init> ()V
```

Comment:
Breaking change.

IIRC this is due to a little known feature in Kotlin's Java interop where it effectively has an implicit `@JvmOverloads` on primary constructors with a single optional argument. Idk why it has that, probably for historical reasons.